src/Symfony/Component/DependencyInjection/ContainerBuilder.php

```diff
@@ -1286,7 +1286,7 @@ public function getAutoconfiguredInstanceof()
      *                                    true to resolve to the actual values of the referenced env vars
      * @param array            &$usedEnvs Env vars found while resolving are added to this array
      *
-     * @return string The string with env parameters resolved
+     * @return string|array The string (or array of strings) with env parameters resolved
```

Comment:
should be "mixed" to be accurate:
`@return mixed The value with env parameters resolved if a string or an array is passed`?

Reply:
Good point, updated.